scripts/s3_media_upload

```diff
@@ -136,6 +187,10 @@ def run_write(sqlite_conn, output_file):
         file_path = to_path(origin, filesystem_id, m_type)
         print(file_path, file=output_file)
 
+        thumbnail_path = to_thumbnail_dir(origin, filesystem_id, m_type)
+        thumbnail_path = os.path.join(thumbnail_path, "*")
+        print(thumbnail_path, file=output_file)
```

Comment:
I wonder if it's more useful to omit the trailing `/*`

Reply:
richvdh has pointed out that hostnames can contain magic characters, so printing out globs would encourage insecure use of shell scripting. Going with a trailing '/' instead.